packages/graphql/test/test-host.ts

```diff
@@ -40,14 +63,47 @@ export async function emitWithDiagnostics(
       "@typespec/graphql": compilerOptions,
     },
   });
+
+  /**
+   * There doesn't appear to be a good way to hook into the emit process and get the GraphQLSchema
+   * that's produced by the emitter. So we're going to read the file that was emitted and parse it.
+   *
+   * This is the same way it's done in @typespec/openapi3:
+   * https://github.com/microsoft/typespec/blame/1cf8601d0f65f707926d58d56566fb0cb4d4f4ff/packages/openapi3/test/test-host.ts#L105
+   */
```

Comment:
@bterlson do you have any other ideas here?

Reply:
This is the way, but open to implementing a better way if you have ideas!